migrations/0004_add_acknowledge_items.down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE issues
+DROP COLUMN acknowledge_by;
```

Comment:
We need to change this to a comma

```suggestion
DROP COLUMN acknowledge_by,
```